tools/travis-test.sh

```diff
@@ -80,6 +80,13 @@ setup_base()
 
 run_test()
 {
+  # meson has a hard dependency on ninja, and we need meson to build
```

Comment:
minor: it'd be good not to repeat the same comment multiple times; can you either remove this or change to something like "see note in `pyproject.toml` for why ninja is installed as a test requirement"?